stock_ext_sst/models/stock_quant.py

```diff
@@ -23,3 +23,16 @@ class StockQuant(models.Model):
     product_id = fields.Many2one(
         auto_join=True,
     )
+
+    def update_stock_move_done_qty(self):
+        stock_moves = self.env['stock.move'].search([
+            ('product_id', '=', self.product_id.id),
+            ('state', 'in', ('confirmed', 'waiting'))
+        ], order='date')
+        for stock_move in stock_moves:
+            if stock_move.picking_type_id.default_location_dest_id.usage == \
+                    'customer':
```

Comment:
@TimLai125 I think the performance could be slightly improved by creating a list of picking_type_id before the `for` loop instead of running the chain of queries for each stock move.  A minor point.